tests/integration/docker/volume.bats

```diff
@@ -48,6 +48,31 @@ setup() {
 	[ $RC -eq 0 ]
 }
 
+@test "Volume - bind-mount a directory" {
+	dir_path="$(pwd)/sharedSpace"
+	test_file="foo"
+	containerPath="/root/sharedSpace"
+	mkdir "$dir_path"
```

Comment:
If this volume test fails at the `docker run` stage for some reason, and the test is then re-run, it will fail on this `mkdir` as the directory will exist from the previous test run. The best solution would probably be to add a `setup()` and `teardown()` and have `dir_path` handled there to ensure a completely clean test environment for each test run.